### ISSUE TITLE
ios support

### ISSUE THREAD

**Issue by hadiyazdi** [stepanhruda/PostgreSQL-Swift#23]
Any possible future ios support?

**Comment by stepanhruda** [comment]
I don't think you can run PostgreSQL on iOS. It doesn't sound like the right tool for the platform anyway. What do you have in mind to use it for?

**Comment by hadiyazdi** [comment]
I want to connect my app to a local postgres database to retrieve some data

**Comment by hadiyazdi** [comment]
My iOS app*

**Comment by stepanhruda** [comment]
Ah ok. Yea, it's not a primary use case, but we might add support for this.

**Comment by mmisesin** [comment]
Any hope I can get this running with an iOS app?
